cmd/internal/ec2/add.go

```diff
@@ -63,6 +72,54 @@ func setupAgentForServer(collectionId string) error {
 	return nil
 }
 
+func askToReconfigure() error {
+	printer.Infof("postman-lc-agent is already present as a systemd service\n")
+	printer.Infof("Helpful commands \n Check status: systemctl status postman-lc-agent \n Disable agent: systemctl disable --now postman-lc-agent \n Check Logs: journalctl -fu postman-lc-agent\n Check env file: cat %s \n Check systemd service file: cat %s \n", envFilePath, serviceFilePath)
+	err := survey.AskOne(
+		&survey.Input{
+			Message: "Over-write old apiKey and collectionId with current values ? (y/n)",
```

Comment:
```suggestion
			Message: "Overwrite old API key and Collection ID with current values? (y/n)",
```

If you switch to "Confirm' I don't think the Y/N will be needed.